jupyterhub/values.yaml

```diff
@@ -51,7 +51,7 @@ hub:
 
 rbac:
   enabled: true
-
+  serviceAccount: ''
```

Comment:
We will actually have a kubernetes `ServiceAccount` object even though the value `rbac.serviceAccount` in `values.yaml` is left blank. I'm thinking that if we name the value `rbac.customServiceAccount` we might indicate the behavior of the setting better. What do you think?

Reply:
i'm fine with it